### PR TITLE
screencap: add German translation

### PR DESCRIPTION
**Pull request by Eclipzze**
Adds a german translation for screencap.md

- [x] The page(s) have at most 8 examples.

### CODE REVIEW

File: pages.de/android/screencap.md/screencap.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Bildschirmfotos
```
Comment (kbdharun) [inline pages.de/android/screencap.md/screencap.md:1]:
```suggestion
# screencap
```
We don't translate the command name across our pages (with a few exceptions like this https://github.com/tldr-pages/tldr/pull/10840/files where the title isn't the command name as our linter doesn't accept symbols)

File: pages.de/android/screencap.md/screencap.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+> Nimmt ein Bildschirmfoto von einem Handy-Display auf
+> Dieser Befehl kann nur über die 'adb shell' benutzt werden
+> Für weitere Informationen: <https://developer.android.com/studio/command-line/adb#screencap>.
+
+- Ein Bildschirmfoto aufnehmen:
+
+`screencap {{path/to/file}}`
```
Comment (kbdharun) [inline pages.de/android/screencap.md/screencap.md:9]:
```suggestion
`screencap {{pfad/zu/datei}}`
```
We translate the path placeholders across our pages, allowing native readers to better understand the examples.

**Review by kbdharun — CHANGES_REQUESTED** [review]
Welcome to tldr and thanks for your contribution.

The directory, file is duplicated. Please fix it so that the path is `pages.de/android/screencap.md`.

You can either do it with `git` locally or else if you want to do it in GitHub itself then click on Edit on this file, then backspace and remove the duplicate directory (before committing add some characters to the file name [this is to prevent conflict with the directory which is named !.md too], then make another commit to fix the filename (removing the characters you added in last step).

File: pages.de/android/screencap.md/screencap.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# screencap
+
+> Nimmt ein Bildschirmfoto von einem Handy-Display auf
+> Dieser Befehl kann nur über die 'adb shell' benutzt werden
```
Comment (gutjuri) [inline pages.de/android/screencap.md/screencap.md:4]:
```suggestion
> Dieser Befehl kann nur über die 'adb shell' benutzt werden.
```

File: pages.de/android/screencap.md/screencap.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# screencap
+
+> Nimmt ein Bildschirmfoto von einem Handy-Display auf
```
Comment (gutjuri) [inline pages.de/android/screencap.md/screencap.md:3]:
```suggestion
> Nimmt ein Bildschirmfoto von einem Handy-Display auf.
```

**Review by gutjuri — CHANGES_REQUESTED** [review]

File: pages.de/android/screencap.md/screencap.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+> Dieser Befehl kann nur über die 'adb shell' benutzt werden
+> Für weitere Informationen: <https://developer.android.com/studio/command-line/adb#screencap>.
+
+- Ein Bildschirmfoto aufnehmen:
```
Comment (pixelcmtd) [inline pages.de/android/screencap.md/screencap.md:7]: As [stated in our Style Guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood), example descriptions are phrased in an imperative mood.

```suggestion
- Nehme ein Bildschirmfoto auf:
```

Comment (gutjuri) [reply pages.de/android/screencap.md/screencap.md:7]: should be "Nimm ein Bildschirmfoto auf" then.

Comment (pixelcmtd) [reply pages.de/android/screencap.md/screencap.md:7]: @gutjuri `de/linux/arecord` already uses "Nehme [...] auf" and I couldn't find anything with "Nimm", so I thought we might wanna standardize on that

File: pages.de/android/screencap.md/screencap.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# screencap
+
+> Nimmt ein Bildschirmfoto von einem Handy-Display auf
+> Dieser Befehl kann nur über die 'adb shell' benutzt werden
+> Für weitere Informationen: <https://developer.android.com/studio/command-line/adb#screencap>.
```
Comment (pixelcmtd) [inline pages.de/android/screencap.md/screencap.md:5]:
```suggestion
> Weitere Informationen: <https://developer.android.com/studio/command-line/adb#screencap>.
```

**Comment by kbdharun** [comment]
It's been a month since the PR was created (and the author hasn't responded yet). As per the maintainer's guide we can take over the PR and apply the changes (I will do it now).

**Review by kbdharun — APPROVED** [review]

**Pull request merged**